app/src/test/java/org/simple/clinic/bp/entry/BloodPressureEntrySheetControllerTest.kt

```diff
@@ -896,4 +896,62 @@ class BloodPressureEntrySheetControllerTest {
 
     verifyNoMoreInteractions(sheet)
   }
+
+  @Test
+  fun `when done button is clicked in update BP entry, then save BP with entered date immediately`() {
+    val systolic = 120.toString()
+    val diastolic = 110.toString()
+    val createdAt = LocalDate.of(1990, 1, 13).atTime(OffsetTime.now(testUtcClock)).toInstant()
```

Comment:
We are using `UserClock` in the `BloodPressureEntrySheetController`, hence change this method to use the `testUserClock` to convert to `Instant`. Like this:

`<date>.toUtcInstant(testUserClock)`

Reply:
Please do the same wherever you are creating the `Instant` from `LocalDate`